css/main.css

```diff
@@ -21,11 +21,14 @@
 
 .exp_the_rush {
   background: url(../img/background-3.jpg) no-repeat center top / cover;
+  width: 1600px;

```

Comment:
this adds scrollbar, remove please

Reply:
@probil, but without fixed width this section can't be pixel perfect

Reply:
Pixel perfect can be verified only on particular screen size.
You can resize your browser to be 1600px instead of making block 1600px width.

Usually designer provides you with PSD for each screen break point (e.g 1600px,  1280px, 960px, 670px, 420px, etc)

Pixel perfects far less important than working and convenient product.

Reply:
My screen is 1440px, how I supposed to use your site? :)

Reply:
> Pixel perfects far less important than working and convenient product.
I thought, that in this case pixel perfect is more important :)